test/evergreen.yml

```diff
@@ -5553,6 +5559,59 @@ buildvariants:
   tasks:
     - name: examples-c-tsan
 
+# This variant runs all possible tasks under TSan but instructs TSan to not report any issues it
+# encounters. This allows us to check whether a TSan change may cause other unexpected failures
+# while also not having to fix every TSan issue discovered. At the conclusion of the TSan work this
+# variant will be deleted.
```

Comment:
What "the conclusion of the TSan work" means is a bit ambiguous. Can we have a ticket created and added to this comment for easier tracking?

Reply:
I've created WT-13256 and added a FIXME in [bc92ac1](https://github.com/wiredtiger/wiredtiger/pull/10839/commits/bc92ac1e03a060b5255bb59bad58e09a2df9235d) 👍